packages/components/pie-divider/src/divider.scss

```diff
@@ -1,22 +1,52 @@
 @use '@justeattakeaway/pie-css/scss' as p;
 
 .c-divider {
-    --divider-bg-color: var(--dt-color-divider-default);
-    --divider-width: 100%;
-    --divider-height: 1px;
+  --divider-bg-color: var(--dt-color-divider-default);
+  --divider-width: 100%;
+  --divider-height: 1px;
+  --divider-label-max-width: 90%;
 
-    margin: 0;
-    border: 0;
-    width: var(--divider-width);
+  margin: 0;
```

Comment:
Just a small suggestion. You could add a rule like below, just to get rid of the background definition for both `c-divider` and the hr when labelled:

```
.c-divider {
    &,
    hr {
        margin: 0;
        border: 0;
        background-color: var(--divider-bg-color);
    }
```

And then you could remove those definitions from here and in the `c-divider--labelled hr` selector?